.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -37,6 +37,22 @@
     </inspection_tool>
     <inspection_tool class="FunctionWithMultipleLoopsJS" enabled="true" level="WARNING" enabled_by_default="true" />
     <inspection_tool class="FunctionWithMultipleReturnPointsJS" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="HtmlUnknownTag" enabled="true" level="WARNING" enabled_by_default="true">
```

Comment:
@MaxKorlaar could you eloborate if these changes are necessary?

Reply:
It is an inspection that's actually enabled by default, warning the user if they're using custom HTML tags not currently in the HTML spec.

Reply:
The same inspection allows you to add custom tags to the list of HTML tags as you can see in the lines below this. Although embed, script and noscript should be built in oddly enough